src/lib/Scenes/ReferenceShow/index.tsx

```diff
@@ -0,0 +1,140 @@
+import { ReferenceShow_show } from "__generated__/ReferenceShow_show.graphql"
+import React from "react"
+import { FlatList, ViewProperties } from "react-native"
+import { createFragmentContainer, graphql } from "react-relay"
+
+import { Artists } from "./Components/Artists"
+import { Artworks } from "./Components/Artworks"
+import { Header } from "./Components/Header"
+import { Location } from "./Components/Location"
+import { Shows } from "./Components/Shows"
+
+interface Props extends ViewProperties {
+  show: ReferenceShow_show
+}
+
+interface State {
+  sections: Array<{
+    type: "location" | "artworks" | "artists" | "shows"
+    data: any
+  }>
+}
+
+export class ReferenceShow extends React.Component<Props, State> {
+  state = {
+    sections: [],
+  }
+
+  componentDidMount() {
+    const { show } = this.props
+    const sections = []
+
+    sections.push({
+      type: "location",
+      data: show.location,
+    })
+
+    sections.push({
+      type: "artworks",
+      data: show.artworks,
+    })
+
+    sections.push({
+      type: "artists",
+      data: show.artists,
+    })
+
+    // TODO: Add shows data
+    sections.push({
+      type: "shows",
+      data: [{ __id: "foo" }],
+    })
+
+    this.setState({ sections })
+  }
+
+  render() {
+    return (
+      <FlatList
+        data={this.state.sections}
+        ListHeaderComponent={<Header />}
+        renderItem={({ item: { data, type } }) => {
+          switch (type) {
+            case "map":
+              return <Location location={data} />
+            case "artworks":
+              return <Artworks artworks={data} />
+            case "artists":
+              return <Artists artists={data} />
+            case "shows":
+              return <Shows shows={data} />
+            default:
+              return null
+          }
+        }}
+        keyExtractor={(item, index) => item.type + String(index)}
+      />
+    )
+  }
+}
+
+export default createFragmentContainer(
+  ReferenceShow,
+  graphql`
+    fragment ReferenceShow_show on Show {
```

Comment:
There's some churn between `ReferenceShow` and `Show` for this container. We'd like to use `Show`, but have to refactor the [existing conflicting definition](https://github.com/javamonn/emission/blob/bdd134aa35b520bbacba0d642400043992c1bb3e/src/lib/Components/Artist/Shows/Show.tsx#L46). I got partway through doing this in this PR but reverted after running into some unrelated issues, thinking I'll follow open a follow up PR to make the naming more consistent.

Reply:
I discussed this with @alloy a bit and he told me about coming to the decision of applying better namespacing to fragments. I'll rename this to something like `LocalDiscoveryShow`, and we can ticket the follow up work to namespace other fragments, e.g.`Show_show` should become something like `ArtistsShow_show`

Reply:
Discussed this a bit more with @l2succes and ended up waffling - the local discovery show is intended to be the canonical showI think, so it seems to make sense to namespace the Artist's `show`. I'm open to discussion though.